examples/save-to-disk/main.go

```diff
@@ -56,23 +72,22 @@ func main() {
 			}
 		}()
 
-		if track.Codec().Name == webrtc.VP8 {
-			fmt.Println("Got VP8 track, saving to disk as output.ivf")
-			i, err := ivfwriter.New("output.ivf")
+		codec := track.Codec()
+		if codec.Name == webrtc.Opus {
+			fmt.Println("Got Opus track, saving to disk as output.opus")
+			i, err := opuswriter.New("output.opus", codec.ClockRate, codec.Channels)
```

Comment:
Mind changing the filename to `output.ogg` (unless that is wrong) just so the user understands.

Reply:
The RFC specifies that the recommanded extension is `.opus`
https://tools.ietf.org/html/rfc7845#section-9
But if you think `.ogg` will be clearer for people, that's fine by me!

Reply:
Nope I am wrong here! thanks for the pushback